scilpy/tractanalysis/fibertube_scoring.py

```diff
@@ -383,6 +383,10 @@ def endpoint_connectivity(blur_radius, centerlines, centerlines_length,
     segment", which is the closest fibertube segment to its before-last
     coordinate.
 
+    IMPORTANT: Streamlines given as input to be scored should be forward-only,
+    which means they are saved so that [0] is the seeding position and [-1] is
```

Comment:
Before, I think the streamline was reversed, i.e. [0] was the last point. It's not the case anymore?

Reply:
Yes. Because I enforce forward_only, it saves the streamline the other way. Given my seeding method and the fact that I track fibertubes, I don't really need to track both ways for now.